### DISCUSSION TITLE
links/

### DISCUSSION THREAD

**Discussion by giscus** [MengZe2l/mengze2l.github.io#3] [General]
# links/

程序更换，友链丢失 想和我做邻居吗？  贵站长可在添加小站的友链后在留言板按如下格式留言，我看到留言且确认贵站已添加小站后会第一时间添加并发邮件通知贵站长  友链格式 12345贵站名称：名称贵站简介：简介贵站链接：链接贵站图片：图片站长邮箱：邮箱   ——————————————名称： 梦泽の日记网址： https://mengze2.cn图标： https://mengze2.cn/fav

https://mengze2.cn/links/

**Comment by LineXic** [comment]
重新申请友链👀
🏠名称LineXic书屋
📰简介念念不忘，必有回响
🛶链接https://linexic.top
🌆图片https://linexic.top/logo.png
📬邮箱linexic@yeah.net

**Reply by MengZe2l** [reply]
已经添加了哦~
